packages/host/app/components/matrix/room-message.gts

```diff
@@ -270,11 +280,23 @@ export default class RoomMessage extends Component<Signature> {
       .join(', ');
   }
 
-  @action patchCard(cardId: string, attributes: Record<string, unknown>) {
+  @action async patchCard() {
```

Comment:
This could be a task instead of an action so that it gets handled (canceled) if the component is destroyed. Maybe best to avoid `async` actions for that reason? We don't have `@action async` anywhere in the code base